### PR TITLE
PIKI as an exceptional authentication level on resource (TAM-141)

### PR DESCRIPTION
**Pull request by urshala**
There is a need such that some resources can only be reserved with
PIKI library card. Even strong authentication shouldn't be able to
reserve such resources. Therefore, add a new authentication level
on resource called PIKI.

Users logged in with PIKI should be able to reserve all resources 
other than the one that requires strong authentication. So, 
PIKI login should be able to reserve resources whose authentication
level is PIKI/Mid/Weak/None.

### CODE REVIEW

File: locale/sv/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -536,11 +536,18 @@ msgstr "Felaktig pk \"{pk_value}\" - objektet existerar inte."
 msgid "Invalid value in filter %(filter)s"
 msgstr "Felaktigt värde i filter %(filter)s"
 
+msgid "You need to login with PIKI library card to reserve this resource."
+msgstr ""
```
Comment (jopesy) [inline locale/sv/LC_MESSAGES/django.po]: "Du måste logga in med en PIKI resursenbibliotekskort för att reservera den här resursen."

File: locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -542,11 +542,18 @@ msgstr "Virheellinen pk \"{pk_value}\" - kohdetta ei ole olemassa."
 msgid "Invalid value in filter %(filter)s"
 msgstr "Virheellinen arvo filtterissä %(filter)s"
 
+msgid "You need to login with PIKI library card to reserve this resource."
+msgstr ""
```
Comment (wiltso) [inline locale/fi/LC_MESSAGES/django.po]: Missing translation, my Finnish is not prefect so please confirm with someone else that this is correct:
`Sinun tulee kirjautua sisään PIKI-kirjastokortilla varataksesi tämän tilan/resurssin.`

File: locale/sv/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -536,11 +536,18 @@ msgstr "Felaktig pk \"{pk_value}\" - objektet existerar inte."
 msgid "Invalid value in filter %(filter)s"
 msgstr "Felaktigt värde i filter %(filter)s"
 
+msgid "You need to login with PIKI library card to reserve this resource."
+msgstr "Du måste logga in med en PIKI resursenbibliotekskort för att reservera den här resursen."
```
Comment (wiltso) [inline locale/sv/LC_MESSAGES/django.po]: Du måste logga in med ett PIKI bibliotekskort för att reservera den här resursen.

Comment (urshala) [reply locale/sv/LC_MESSAGES/django.po]: Updated it.

File: locale/fi/LC_MESSAGES/django.po (hunk 2 above)
Comment (urshala) [reply locale/fi/LC_MESSAGES/django.po]: Updated it.

**Review by wiltso — COMMENTED** [review]
Looks good to me :tada:

**Pull request merged**